Dockerfile

```diff
@@ -1,22 +1,8 @@
-# This is a multi-stage Dockerfile and requires >= Docker 17.05
-# https://docs.docker.com/engine/userguide/eng-image/multistage-build/
-FROM gobuffalo/buffalo:v0.12.7 as builder
+FROM gobuffalo/buffalo:v0.12.7
 
 RUN mkdir -p $GOPATH/src/github.com/cpjudge/cpjudge_webserver
 WORKDIR $GOPATH/src/github.com/cpjudge/cpjudge_webserver
 
-ADD . .
-RUN dep ensure
```

Comment:
How does it install the required dependencies? We are mounting the current directory, thats why it is not required?

Reply:
And if this is the case, we should add Installing go dependencies in README.md by running dep ensure.

Reply:
My bad. its there.